.github/workflows/backport-assistant.yml

```diff
@@ -0,0 +1,28 @@
+# This action creates downstream PRs for PRs with backport labels defined.
+# See docs here: https://github.com/hashicorp/backport-assistant
+
+name: Backport Assistant Runner
+
+on:
+  # TODO: Figure out if this should be `pull_request_target` if we have forks.
+  pull_request:
+    types:
+      - closed
+      - labeled
```

Comment:
Am I correct in assuming that the `labeled` event would trigger this whenever someone adds a label to the PR? If that is true, it seems that it would run prior to being merged. Is this what we want? If that's true, it seems to be at odds with the `closed` event above it. Can you confirm the intended behavior here? 

I did find this in [the docs for the `label` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#label):
>  If you want to run your workflow when a label is added to or removed from an issue, pull request, or discussion, use the labeled or unlabeled activity types

This seems to indicate that this workflow would run when a label is added which could be pre-merge.

Reply:
Good question. My understanding is that this is intended, with the usage being that you could add the backport labels post-merge and they would still get applied. Line 15 below (`if: github.event.pull_request.merged`) would guard against the workflow running pre-merge.